fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/CreateNonRdfSourcePersisterTest.java

```diff
@@ -98,12 +103,12 @@
 
     private static final Long EXTERNAL_CONTENT_SIZE = 526632l;
 
-    private final NonRdfSourcePersister persister = new NonRdfSourcePersister();
+    private CreateNonRdfSourcePersister persister;
 
     @Before
     public void setUp() throws Exception {
```

Comment:
Maybe add a test for what happens if you try to create a resource that already exists?

Reply:
Checking for exisitng resources should be handled at a higher level.